test/functional/fixtures/api/es-next/request-hooks/testcafe-fixtures/add-remove-request-hook.js

```diff
@@ -0,0 +1,68 @@
+import { RequestHook } from 'testcafe';
+import { RequestFilterRule } from 'testcafe-hammerhead';
+import path from 'path';
+
+const ResultPromise = require(path.resolve('./lib/utils/result-promise'));
+const pageUrl       = 'http://localhost:3000/fixtures/api/es-next/request-hooks/pages/index.html';
+
+class TestRequestHook extends RequestHook {
+    constructor (name) {
+        super();
+
+        this.name              = name;
+        this.requestFilterRule = new RequestFilterRule(pageUrl);
+
+        this.onResponseCallCountInternal = 0;
+    }
+
+    get onResponseCallCount () {
+        return ResultPromise.fromFn(async () => this.onResponseCallCountInternal);
+    }
+
+    getRequestFilterRules () {
+        return [this.requestFilterRule];
```

Comment:
don't import `RequestFilterRule` from `testcafe-hammerhead`. Use `return [pageUrl]`. This is the way how users will use it.

Reply:
It's difficult to use simple types instead of `RequestFilterRule`